libweasyl/libweasyl/staff.py

```diff
@@ -54,17 +52,16 @@
 """ Purely cosmetic group for users who contribute to site development. """
 
 
-def _init_staff(staff_config_path):
+def _init_staff(staff_config_dict):
```

Comment:
Thoughts on having this accept kwargs `directors`, `technical`, `admins`, `mods`, `developers`, each with default `frozenset()`? @Weasyl/dev